swlor2_2da/loadscreens.2da

```diff
@@ -1,7 +1,25 @@
 2DA V2.0
 
            Label         ScriptingName                 BMPResRef       TileSet     StrRef
-0          Random        ****                          ls_tcn_01            ****        ****
+0          Random        ****                          ****            ****        ****
```

Comment:
GitHub is reporting a merge conflict on this file. Can you double check and get that resolved? Reach out if you need help.